docs/sources/structure/topic-types/visualization/index.md

```diff
@@ -0,0 +1,121 @@
+
+A visualization topic includes the following elements:
+
+- **Topic title:** A visualization topic title is the same as the name of the visualization as it's written in the UI (for example, "Time series"). Don't add the word "panel" or "visualization" to the title.
+- **Introduction:** Include an introduction that explains what the visualization type can help the user do.
+- **Main image:** Add a clear screenshot depicting the visualization displaying a typical use case.
+- **Body:** The body may include task/how-to information as needed. The body must include the following sections:
+
+  - **Supported data formats:** Provide guidance about which data formats are supported by the visualization type, with example use cases.
+  - **[Section] options:** For each drop-down section of options in the edit panel, add a section in the topic with the name of the UI section, followed by the word "options." For example, if a drop-down section of options is called "Axis" in the UI, include a section in the topic called "Axis options". This doesn't include the sections linked from the **Other options** section.
+  - **Other visualization options:** Include a section for links to the documentation for common UI options:
+
+    - Panel options
+    - Standard options
+    - Thresholds
+    - Legend
+    - Data links
+    - Field overrides
+    - Value mappings
+
+![Visualization structure](/media/docs/writers-toolkit/visualization-topic-example-ann-2.png)
+
+### Widget topic structure
+
+_Widgets_ are visualizations that don't require a data source. These visualizations have far fewer options and don't need as much explanation. (For visualizations that require a data source, refer to [Visualization topic structure](#visualization-topic-structure).) A widget topic includes the following elements:
+
+- **Topic title:** A widget topic title is the same as the name of the widget as it's written in the UI (for example, "Time series"). Don't add the word "panel", "visualization", or "widget" to the title.
+- **Introduction:** Include an introduction that explains what the visualization type can help the user do.
+- **Main image:** Add a clear screenshot depicting the visualization displaying a typical use case.
+- **Body:** The body may include task/how-to information as needed. The body must include the following sections:
+
+  - **[Section] options:** For each drop-down section of options in the edit panel, add a section in the topic with the name of the UI section, followed by the word "options." For example, if a drop-down section of options is called "Axis" in the UI, include a section in the topic called "Axis options".
+
+### Visualization naming conventions
+
+When writing about visualizations (or widgets), use:
+
+- Noun form
+- Lower case, unless at the beginning of a sentence
+
+For example:
+
+```
```

Comment:
nit: You might want to use `markdown` as the code fence style to get syntax highlighting if the examples become more Markdown-y.